### ISSUE TITLE
Test support for large files

### ISSUE THREAD

**Issue by blairdrummond** [StatCan/minio-deploy#3]
See how a large file gets loaded onto PVCs which are smaller than the file itself.

**Comment by blairdrummond** [comment]
Test 1: Failed

Went over the storage cap. Looks like it went to 13gb though, so it probably fragmented the file across PVCs but ran out of total space.

![file-size-cap](https://user-images.githubusercontent.com/10801138/83041940-06f69b80-a00f-11ea-9638-6f9c3a9ea381.png)

**Comment by blairdrummond** [comment]
@zachomedia Did we patch minio manually? I'm not seeing any recent commits to this repo. We did increase the cap on `minimal` right?

**Comment by blairdrummond** [comment]
@zachomedia What's the expected downtime if the tenant was upgraded?

**Comment by blairdrummond** [comment]
**Results:**
  1. Filling up Minio is bad, and we did it more than once.
  2. To resolve (1), we resized our instances to be quite large (1tb each?), so testing files larger than the PVC size (to check if minio shards the data across PVC boundaries) is no longer applicable (unless we upload a >1TB file)

**Conclusion:**

Uploading large files is not an issue so long as they're smaller than the PVCs; this doesn't seem like a huge issue. More interesting consideration will be monitoring Minio space usage and possibly implementing an auto-scale-up should we encounter issues again.
